libs/paths/Realpath.mli

```diff
@@ -20,16 +20,13 @@
    or directories. You cannot use this for something which does not
    yet exist. Use the Fpath library for that instead.
 
-   Also note that even though it's tempting to use Rpath.t inside your
+   Also note that even though it's tempting to use Realpath.t inside your
    program, because they provide a canonical representation of a path,
    you should prefer in general to use Fpath.t because users want
    error messages, findings, etc. that contain paths derived from
    what was passed on the command line, or Ppath.t if users want
    findings relative to the root of their project (and not real paths like
    /home/pad/my/long/project/foo/bar).
-
-   The name of the module imitates Fpath.ml, and Ppath.ml, but use Rpath.ml
```

Comment:
See, it was done on purpose!

Reply:
I have no doubt it was done on purpose but it is and should remain an uncommon call due to its cost, and uncommon things should have longer, more descriptive names.